### PR TITLE
MatthewKuKanich/Radar_Scanner_1.0

### PR DESCRIPTION
**Pull request by MatthewKuKanich**
# Application Submission

- The Flipper Zero Radar Scanner is an application designed for the Flipper Zeros GPIO capabilities, utilizing the RCWL-0516 Microwave Radar module to detect the presence and movement of living beings or people. This powerful radar module is capable of detecting movement at a range of approximately 8 meters, depending on the environment, and can even detect humans through walls, ceilings, and floors up to 5-6 meters away.
- Easy Hardware Setup: Connect the RCWL-0516 module to your Flipper Zero by supplying 5 volts to the VIN pin with ground connected. The OUT pin of the module should be connected to Pin 7 on the Flipper Zero. Make sure to enable the "5V on GPIO" option when powering the radar module from the Flipper Zero.
- Bi-Directional Detection: The RCWL-0516 module is bi-directional, meaning it can detect movement from both the front and back sides. The front side with components is the primary detection side and should be oriented away from you. You can modify the detection range by adding metal shielding to focus detection where the Flipper is pointing.
- Adjustable Sensitivity: If the sensitivity is too high, you can solder a resistor to the module's pad to reduce it, allowing you to fine-tune the detection sensitivity to your needs.
- User-Friendly Interface: The app starts in an "inactive" or "standby" state. Press the OK button to activate it. The Flipper display will show "Active," and the app will begin monitoring for movement.

- Real-Time Feedback:
    When movement is detected, the text "No presence" will change to "Presence detected."
    An alarm will sound to alert you.
    The LED will change from green ("clear") to red ("detected").
    The Flipper will vibrate twice to notify you.

- Muted Mode: Press the Down button to enter "muted" mode. In this mode, detection still triggers text changes, LED status, and vibrations, but there is no sound. You can toggle this mode on and off.

# Extra Requirements 

- The Radar Scanner app requires the RCWL-0516 Microwave Radar Module

# Author Checklist (Fill this out)

- [X] I've read the [contribution guidelines](../blob/HEAD/documentation/Contributing.md) and my PR follows them
- [X] I own the code I'm submitting or have code owner's permission to submit it
- [x] I [have validated](../blob/HEAD/documentation/Contributing.md#validating-manifest) the manifest file(s)

# Reviewer Checklist (Don't fill this out)

- [x] Bundle is valid
- [x] There are no obvious issues with the source code
- [x] I've ran this application and verified its functionality

### CODE REVIEW

**Comment by doomwastaken** [comment]
`Error: /tmp/tmpx9bphw0v/code/radar_scanner.c:14:10: fatal error: assets_icons.h: No such file or directory
   14 | #include <assets_icons.h>`
 Build fails, verify that everything is uploaded correctly and I will re-run

**Comment by MatthewKuKanich** [comment]
> `Error: /tmp/tmpx9bphw0v/code/radar_scanner.c:14:10: fatal error: assets_icons.h: No such file or directory 14 | #include <assets_icons.h>` Build fails, verify that everything is uploaded correctly and I will re-run

Apologies, I had several unused imports. I removed them all and I re-tested with a clean clone of both the firmware and my app, and it builds successfully for me.

**Comment by MatthewKuKanich** [comment]
I'm not sure why the same error appears. I removed ```#include <assets_icons.h>```. Did it not clone the latest commits from https://github.com/MatthewKuKanich/flipper-radar ?

**Comment by doomwastaken** [comment]
You also need to change the commit SHA in the manifest, currently it is building https://github.com/MatthewKuKanich/flipper-radar/commit/87faeb83d73a1a8518e5c8dbc8180783a7be3c1c

File: applications/GPIO/radar_scanner/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/MatthewKuKanich/flipper-radar.git
+    commit_sha: 87faeb83d73a1a8518e5c8dbc8180783a7be3c1c
```
Comment (doomwastaken) [inline applications/GPIO/radar_scanner/manifest.yml:5]: You need to update this, also this is what you will be updating via new pull request when you are updating app version

Comment (MatthewKuKanich) [reply applications/GPIO/radar_scanner/manifest.yml]: Thank you! That makes total sense, updating the hash went over my head but it's how you do versioning. Thanks for letting me know, it's my first app so I'm still learning. It should be set to the latest commit now

**Comment by doomwastaken** [comment]
Also mobile markdown version is limited and it does not support backticks - '

**Review by MatthewKuKanich — COMMENTED** [review]
I hope this is the last hiccup, I'm sorry for taking so much of your time

**Comment by doomwastaken** [comment]
I will get to you shortly, as far as I understand now issue was with not connected to current README.md, but with 
https://github.com/MatthewKuKanich/flipper-radar/blob/main/README.md
This does not make sense to me, since you added custom README.md file along the application manifest. Let me verify this and get back to you

**Comment by doomwastaken** [comment]
Clarified it once again, you only need manifest file in the pull request. Readme.md will be taken from the repository specified in manifest. 

Backticks in your case are from monospace fonts. 
```Like this```

**Comment by MatthewKuKanich** [comment]
I removed load from file and typed it out in basic formatting to make sure it goes through. On the next version release, I can make it look better and load from file

**Review by doomwastaken — APPROVED** [review]

**Pull request merged**